docker-compose.yml

```diff
@@ -1,18 +1,17 @@
 services:
   piper:
     build: .
-    container_name: textymcspeechy-piper
-    hostname: textymcspeechy-piper
+    container_name: ${CONTAINER_NAME:-textymcspeechy-piper}
+    hostname: ${CONTAINER_NAME:-textymcspeechy-piper}
```

Comment:
I am not sure what the hostname is used for, I did not find any issues with keeping it like this, and I did not see it used anywhere inside the docker. Since we use just one docker, and not multiple, I think it is irrelevant.